src/schemas/ui-view-schema.json

```diff
@@ -17,9 +17,16 @@
             "name": {
```

Comment:
We should probably remove `name` from here and the TypeScript type since it is no longer used.

Reply:
Got it, will scrub `name`. Should I also scrub the deprecated activity `type`?